sources/install/package_base.sh

```diff
@@ -396,6 +396,16 @@ function post_install() {
     if [ -d "/root/Desktop" ]; then rm -r /root/Desktop; fi
 }
 
+function uploader() {
```

Comment:
Missing uploader() function call (misc package ?)

Reply:
It missing function call, you need to call your function in the package_misc function